Common/src/main/resources/default_configs/example.json

```diff
@@ -0,0 +1,30 @@
+{
+  "mod_id": "", #add mod_id
+  "provided_stones": [
+    {
+	  "id": "modid_itemid", #Don't add ":"

```

Comment:
That's not what the ID is. The ID is an identifier for the stone. It should be unique. It has nothing to do with the mod/item ID; that's just a convention I follow to guarantee uniqueness